disttask/framework/dispatcher/dispatcher.go

```diff
@@ -65,6 +69,12 @@ type dispatcher struct {
 	task     *proto.Task
 	logCtx   context.Context
 	serverID string
+
+	// for HA
+	liveNodes    []*infosync.ServerInfo
```

Comment:
Please add detailed explain for liveNodeIntv,
And change it to `liveNodeCheckInterval`